Calculate expression added to modeler tools (#8550)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/modelertools.rst

      :class: toc_columns

.. _qgiscalculateexpression:

Calculate expression 
--------------------

> Calculate the results of a QGIS expression and make 

Reviewer: ```suggestion
|334|

Calculates the result of a QGIS expression and makes 
```
and mention substitution at the end of file.

the result available for use in other parts of the model.

Parameters
..........

.. list-table::
   :header-rows: 1
   :widths: 20 20 20 40

   * - Label
     - Name
     - Type
     - Description

>    * - **Field**

Reviewer: ```suggestion
   * - **Input**
```

     - ``INPUT``
     - [expression]
     - Expression to calculate 

Outputs
.......

.. list-table::
   :header-rows: 1
   :widths: 20 20 20 40

   * - Label
     - Name
     - Type
     - Description

>    * - **Output**
>      - ``Value``

Reviewer: ```suggestion
   * - **Value**
     - ``OUTPUT``
```
I think there is a switch here (https://github.com/qgis/QGIS/pull/54358/files#diff-dc398bcb04bd91d852555624aeccefce91fe509dc7ad879ba12ce978d67fa238R82)
